core/object.go

```diff
@@ -300,92 +301,108 @@ type (
 var TYPES = map[*string]*Type{}
 var TYPE Types
 
-func RegRefType(name string, inst interface{}) *Type {
-	t := &Type{name: name, reflectType: reflect.TypeOf(inst)}
+func RegRefType(name string, inst interface{}, doc string) *Type {
+	if doc != "" {
+		doc = "\n  " + doc
+	}
+	meta := MakeMeta(nil, "(Concrete reference 'Type')"+doc, "1.0")
```

Comment:
Not sure I like quoting here. How about just `"(Concrete reference type)"` (and `"(Concrete type)"` below)?